### ISSUE TITLE
[PROBABLY BUG] Inodes overconsumption on 2.0

### ISSUE THREAD

**Issue by Fell-x27** [input-output-hk/cardano-sl#3985]
1)  Install cardano-sl
2) connect it to a mainnet or testnet in a foreground
3) interrupt it with CTRL+C
4) try to connect again

Expected behaviour: synchronization with very low inodes consumption.
Real behaviour: very fast (!!!) synchronization, that eats all your inodes.

If you cant reproduce it, repeat stages 2 and 3 for couple times.

Removing DB directory doesn't help - cardano creates it again and eats all your inodes...again.
Removing whole cardano-sl and reinstalling are helpfull. Until next accidental interruption.

**Comment by erikd** [comment]
Does this explain the symptoms you see: https://iohk.zendesk.com/hc/en-us/articles/360014432734-Daedalus-0-12-0-Optimized-block-storage

**Comment by Fell-x27** [comment]
Yes, but no. Because it says, that this problems are about upgrading from an old version to a new one. 
But you can reproduce it on the new version too. Yes, I understand why it happens, but I think, that regular users, probably, just dont care about this "technical stuff". They just install Daedalus, launch it, look at this, stop it to launch again later, because of time consumption. And...they got this problem.  

Many people got it. I think, it would be great, if Daedalus check and fix it by someway without any outside manipulations.
